### PR TITLE
remove null propagation with unity object

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change cleans up null propagation and null coalescence usages with Unity objects. 

Recommend taking for 2.6 as it should resolve potential race condition bugs based on when Unity objects actually get set to null.

### CODE REVIEW

File: Assets/MRTK/SDK/Features/UX/Scripts/Utilities/FeaturesPanelVisuals.cs [hunk 1]
```diff
@@ -27,8 +27,15 @@ void Start()
         {
             profilerButton.IsToggled = (CoreServices.DiagnosticsSystem?.ShowProfiler).GetValueOrDefault(false);
             handRayButton.IsToggled = PointerUtils.GetPointerBehavior<ShellHandRayPointer>(Handedness.Any, InputSourceType.Hand) != PointerBehavior.AlwaysOff;
-            handMeshButton.IsToggled = (CoreServices.InputSystem?.InputSystemProfile?.HandTrackingProfile.EnableHandMeshVisualization).GetValueOrDefault(false);
-            handJointsButton.IsToggled = (CoreServices.InputSystem?.InputSystemProfile?.HandTrackingProfile.EnableHandJointVisualization).GetValueOrDefault(false);
+
+            MixedRealityHandTrackingProfile handProfile = null;
+            if ((CoreServices.InputSystem != null) &&
+                (CoreServices.InputSystem.InputSystemProfile != null))
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/Utilities/FeaturesPanelVisuals.cs:33]: You should still be able to use null propagation on `CoreServices.InputSystem` if you wanted to simplify, but either way works!

```suggestion
            if (CoreServices.InputSystem?.InputSystemProfile != null)
```

File: Assets/MRTK/SDK/Features/UX/Scripts/Utilities/ToggleHandVisualisation.cs [hunk 2]
```diff
@@ -32,7 +32,15 @@ public void OnToggleHandMesh()
         /// </summary>
         public void OnToggleHandJoint()
         {
-            MixedRealityHandTrackingProfile handTrackingProfile = CoreServices.InputSystem?.InputSystemProfile?.HandTrackingProfile;
+            MixedRealityHandTrackingProfile handTrackingProfile = null;
+
+            if ((CoreServices.InputSystem != null) &&
+                (CoreServices.InputSystem.InputSystemProfile != null) &&
+                (CoreServices.InputSystem.InputSystemProfile != null))
```
Comment (MaxWang-MS) [inline Assets/MRTK/SDK/Features/UX/Scripts/Utilities/ToggleHandVisualisation.cs:39]: This line seems to be redundant.

Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/Utilities/ToggleHandVisualisation.cs:39]: Same nit as previous comment, plus the existing third case looks to be a duplicate of the second:

```suggestion
            if (CoreServices.InputSystem?.InputSystemProfile != null)
```

Comment (davidkline-ms) [reply Assets/MRTK/SDK/Features/UX/Scripts/Utilities/ToggleHandVisualisation.cs:39]: haha! great catch!

**Review by MaxWang-MS — APPROVED** [review]

**Review by keveleigh — APPROVED** [review]

**Pull request merged**